util.py

```diff
@@ -53,6 +54,7 @@ def find_closest_servers(self, location, servers_to_search = None):
             best_servers.append(server_dict)
         best_servers.sort(key=self.get_distance_key)
     return best_servers
+"""
```

Comment:
i thought find_closest_servers() was replaced by function find_closest_servers_with_ip(). but seems volley is still using it. will delete the comment.